iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java

```diff
@@ -63,10 +63,12 @@ public static Configuration toHubConfiguration(final DeploymentServiceModel mode
 
         // Required labels
         labels.put(PACKAGE_TYPE_LABEL, model.getPackageType().toString());
-        labels.put(CONFIG_TYPE_LABEL, model.getConfigType().toString());
         labels.put(DEPLOYMENT_NAME_LABEL, model.getName());
         labels.put(DEPLOYMENT_GROUP_ID_LABEL, deploymentGroup.getId());
         labels.put(RM_CREATED_LABEL, Boolean.TRUE.toString());
+        if (!StringUtils.isBlank(model.getConfigType().toString())){
+            labels.put(CONFIG_TYPE_LABEL, model.getConfigType().toString());
+        }
```

Comment:
Can you put empty in else block?

Reply:
Empty string causes exception, so we has to add the if/else block.